clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/security/DcosCredentialMap.groovy

```diff
@@ -0,0 +1,19 @@
+package com.netflix.spinnaker.clouddriver.dcos.security
+
+class DcosCredentialMap {
```

Comment:
Just not a usual convention I guess - usually if you have "Map" in the class name it is assumed to implement the java Map interface.